justfile

```diff
@@ -223,29 +215,20 @@ test *args: start-dependencies create-aurora-ndc-metadata
 
 # re-generate the NDC metadata configuration file
 generate-chinook-configuration: build start-dependencies
-  ./scripts/archive-old-ndc-metadata.sh '{{POSTGRES_V1_CHINOOK_NDC_METADATA}}'
-  ./scripts/generate-chinook-configuration.sh 'ndc-postgres' '{{POSTGRESQL_CONNECTION_STRING}}' '{{POSTGRES_V1_CHINOOK_NDC_METADATA}}'
-  ./scripts/generate-chinook-configuration.sh 'ndc-postgres' '{{CITUS_CONNECTION_STRING}}' '{{CITUS_V1_CHINOOK_NDC_METADATA}}'
-  ./scripts/generate-chinook-configuration.sh 'ndc-postgres' '{{COCKROACH_CONNECTION_STRING}}' '{{COCKROACH_V1_CHINOOK_NDC_METADATA}}'
-
-  ./scripts/archive-old-ndc-metadata.sh '{{POSTGRES_V2_CHINOOK_NDC_METADATA}}'
-  ./scripts/generate-chinook-configuration.sh 'ndc-postgres' '{{POSTGRESQL_CONNECTION_STRING}}' '{{POSTGRES_V2_CHINOOK_NDC_METADATA}}'
-  ./scripts/generate-chinook-configuration.sh 'ndc-postgres' '{{CITUS_CONNECTION_STRING}}' '{{CITUS_V2_CHINOOK_NDC_METADATA}}'
-  ./scripts/generate-chinook-configuration.sh 'ndc-postgres' '{{COCKROACH_CONNECTION_STRING}}' '{{COCKROACH_V2_CHINOOK_NDC_METADATA}}'
+  ./scripts/archive-old-ndc-metadata.sh '{{POSTGRES_V3_CHINOOK_NDC_METADATA}}'
```

Comment:
It'd be good to leave some sort of marker that this is a place which should do something for all currently supported versions and not just the lastest.

Reply:
This is an issue in general. When we reduce our set of supported versions to just the latest it becomes harder to add a new version to that set, because you have to rediscover the parts that are different between the latest version and the full set of supported legacy versions.

Reply:
We will probably have to reconsider the idea of a "version" as soon as we split the configuration into multiple files.

Reply:
Even then, knowing which parts of the supporting dev code historically had to take older versions into account is probably a good baseline understanding IMO.